packages/rainbowkit/src/wallets/Wallet.ts

```diff
@@ -46,14 +45,16 @@ type RainbowKitConnector<C extends Connector = Connector> = {
   };
 };
 
-export type Wallet<C extends Connector = Connector> = {
+export type Wallet = {
   id: string;
   name: string;
+  rdns?: string;
   shortName?: string;
   iconUrl: string | (() => Promise<string>);
   iconAccent?: string;
   iconBackground: string;
   installed?: boolean;
+  ready?: boolean;
```

Comment:
Do we need this here; shouldn't we be fetching this from the Wagmi connector upstream?

Reply:
Nope not needed. Was trying to add my own `ready` since wagmi no longer supports that. We're using `installed` instead.